src/main/resources/com/ibm/appscan/jenkins/plugin/Messages.properties

```diff
@@ -9,7 +10,8 @@ error.checking.results=An error occurred checking the results of analysis. The s
 error.invalid.template.file=Invalid scan template file.
 error.no.creds=Required. You can add IBM ASoC credentials on the <a target="_blank" href="{0}">Credentials page</a>. 
 error.running.scan=Failed to run the security scan: {0}
-error.token.expired=Unable to authenticate. Check your credentials on the <a target="_blank" href="{0}">Credentials page</a>. 
+error.token.expired=Unable to authenticate. Check your credentials on the <a target="_blank" href="{0}">Credentials page</a>.
+error.noncompliant.issues=The scan result contains non compliant issues with respect to to the selected policies.  
```

Comment:
Can you update this string to "... with respect to the selected application's policies"?